### PR TITLE
Write `cypress` test for `solution tab`

### PR DESCRIPTION
**Pull request by Aritra8438**
# Description

This PR adds `cypress` test for `solution tab`.

Fixes: #7596 

# Checklist:

- [x] The code follows the [coding guidelines](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) of omegaUp.
- [x] The tests were executed and all of them passed.
- [x] If you are creating a feature, the new tests were added.
- [x] If the change is large (> 200 lines), this PR was split into various Pull Requests. It's preferred to create one PR for changes in controllers + unit tests in PHPUnit,  and then another Pull Request for UI + tests in Jest, Cypress or both.
Signed-off-by: Aritra Majumder <aritramajumder8438@gmail.com>

### CODE REVIEW

**Comment by Aritra8438** [comment]
This PR needs 
- [ ] #7788 

to be merged.

File: cypress/e2e/problem_creator.cy.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    cy.clearLocalStorage();
+    cy.visit('/');
+  });
+
+  it('Should write and verify the problem statement', () => {
+    const loginOptions: LoginOptions = {
+      username: 'user',
+      password: 'user',
+    };
+    cy.login(loginOptions);
```
Comment (carlosabcs) [inline cypress/e2e/problem_creator.cy.ts:15]: If you are not going to reuse the `loginOptions`, just pass the object directly here.

File: cypress/e2e/problem_creator.cy.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      '<h1>Previsualización</h1>\n\n<p>Hello omegaUp!</p>',
+    );
+  });
+
+  it('Should write and verify the problem solution', () => {
+    const loginOptions: LoginOptions = {
+      username: 'user',
+      password: 'user',
+    };
+    cy.login(loginOptions);
```
Comment (carlosabcs) [inline cypress/e2e/problem_creator.cy.ts:35]: Same here.

**Review by carlosabcs — COMMENTED** [review]
Minor fixes but looks good.

File: cypress/e2e/problem_creator.cy.ts (hunk 2 above)
Comment (Aritra8438) [reply cypress/e2e/problem_creator.cy.ts:34]: Done!

**Review by carlosabcs — APPROVED** [review]
LGTM

**Review by carlosabcs — APPROVED** [review]
LGTM

**Comment by Aritra8438** [comment]
@heduenas PTAL!

**Pull request merged**